release/release_workflow.wdl

```diff
@@ -0,0 +1,280 @@
+task do_release {
+   # Repo to release
+   String repo
+   # Version number that should be released
+   String releaseV
+   # Next version number after this released is done (in preparation for next release)
+   String nextReleaseV
+   # Command that will update the appropriate file for the current release
+   String updateVersionForCurrentRelaseCommand
+   # Command that will update the appropriate file for the next release
+   String updateVersionForNextRelaseCommand
+   # A command to optionnally create a fat jar
+   String? assembleJarCommand
+   
+   # Commands that will update previously released/pubslished dependencies in this repo
+   Array[String] dependencyCommands = []
+   
+   # When true, nothing will be pushed to github, allows for some level of testing
+   Boolean dryRun = false
+   
+   # Can be swapped out to try this on a fork
+   String organization = "broadinstitute"
+    
+   command {
+     set -e
+     set -x 
+     
+     # Clone repo and checkout develop
+     git clone https://github.com/${organization}/${repo}.git
+     cd ${repo}
+     git checkout develop
+     git pull --rebase
+
+     # Extract current version  
+     currentV=$(grep "git.baseVersion " build.sbt | cut -d "=" -f 2 | tr -d \"\,[:space:])
+
+     echo "Updating ${repo} to ${releaseV}"
+     ${updateVersionForCurrentRelaseCommand}
+     
+     echo "Update dependencies"
+     ${sep='\n' dependencyCommands}
```

Comment:
Hmm, although Cromwell is already a command injector, this feels a bit too command injectory, even for me...

Is there no way of expressing this as prerequisite WDL tasks?